lib/project/ext/ui_table_view_controller.rb

```diff
@@ -0,0 +1,13 @@
+class UITableViewController
+  # initWithStyle is called before viewDidLoad, if we use the stylesheet in
+  # the table_data, we need to set the stylesheet earlier.
+  alias :original_initWithStyle initWithStyle
```

Comment:
@jamonholmgren thanks for the keen :eyes: - I updated to match the rest of redpotion which seems to be `alias :original :changed`